### ISSUE TITLE
INFO[0028] cannot get table tableName struct: cannot read trigger: sql: Scan error on column index 5: unsupported Scan, storing driver.Value type <nil> into type *time.Time 

### ISSUE THREAD

**Issue by sahar1454** [Percona-Lab/mysql_random_data_load#11]
Hi,

I get the following error when trying to insert 1000 random rows to any of my tables that have triggers defined on them:
INFO[0028] cannot get table tableName struct: cannot read trigger: sql: Scan error on column index 5: unsupported Scan, storing driver.Value type <nil> into type *time.Time

**Comment by guriandoro** [comment]
Hi Sahar!

I believe this is the same case as https://github.com/Percona-Lab/mysql_random_data_load/issues/9.

As mentioned by Carlos, can you please try with latest 0.1.7 and see if it fixes it?

```
Could you please try the latest version? (0.1.7)
https://github.com/Percona-Lab/mysql_random_data_load/releases/tag/0.1.7
```

Let us know how it goes. If not, please send us the following:

mysql_random_data_load version used
`SHOW CREATE TABLE tableName;`
`SHOW INDEXES FROM tableName;`
`SHOW TRIGGERS LIKE 'tableName';`
mysql version used (`SHOW VARIABLES LIKE '%version%';`)


Thanks!

**Comment by percona-csalguero** [comment]
Fixed in 0.1.8
